js/letters.js

```diff
@@ -15,8 +15,24 @@ var counter = {
   "y" : 0,  "z" : 0
 }
 
-function countLetters(counter, sample_text){
-  // FIX ME
+function countLetters(counter, text){
+  
+  // base case 
+  if (text.length === 0) {
+    return counter;
+  }
+
+  // convert string to lowercase
+  let newText = text.toLowerCase();
+
+  let targetChar = newText.slice(0, 1);
+
+  if ([targetChar]) {
```

Comment:
The square brackets are not necessary in this situation and will actually always return true here. 

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.

Reply:
So this instead? This is passing the test.

```javascript
if (counter.hasOwnProperty(targetChar)) {
```